R/dependencies.R

```diff
@@ -1709,8 +1709,18 @@ renv_dependencies_report <- function(errors) {
 
 renv_dependencies_eval <- function(expr) {
 
-  # create environment with small subset of symbols
-  syms <- c("list", "c")
+  # create environment with small subset of "safe" symbols, that
+  # are commonly used for chunk expressions
+  syms <- c(
+    "list", "c",
+    "::", ":::", "$", "@", "[", "[[",
```

Comment:
Not sure it's worth `::` and `:::` if you don't otherwise support a bigger set of base functions.